datastore/google/cloud/datastore/transaction.py

```diff
@@ -234,3 +244,20 @@ def commit(self):
         finally:
             # Clear our own ID in case this gets accidentally reused.
             self._id = None
+
+    def put(self, entity):
+        """Ensures the transaction is not marked readonly and calls Batch.put
+
+        This method calls :meth:`put` in the superclass, `Batch`, as long as
```

Comment:
This is user-facing documentation, so it's "inappropriate" to be discussing implementation details.